LoopbackGuideSkeleton/build.gradle

```diff
@@ -40,4 +40,4 @@ android {
         debug.setRoot('build-types/debug')
         release.setRoot('build-types/release')
     }
-}
+}
```

Comment:
You appear to be removing the newline from the end of the file. Please don't do that.